src/EditorFeatures/CSharpTest/MoveDeclarationNearReference/MoveDeclarationNearReferenceTests.cs

```diff
@@ -295,7 +295,8 @@ static void Main(string[] args)
         public async Task TestMissingInHiddenBlock1()
         {
             await TestMissingInRegularAndScriptAsync(
-@"class Program
+@"#line default
+class Program
```

Comment:
existing tests should not be updated.

Reply:
Agree, but due to time restrictions outside of currently doable scope. Will try to revisit at the end of the internship.